src/status_im2/contexts/wallet/send/select_address/view.cljs

```diff
@@ -0,0 +1,86 @@
+(ns status-im2.contexts.wallet.send.select-address.view
+  (:require [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [status-im2.contexts.wallet.send.select-address.style :as style]
+            [react-native.safe-area :as safe-area]
+            [quo2.core :as quo]
+            [utils.re-frame :as rf]
+            [utils.i18n :as i18n]
+            [status-im2.constants :as constants]
+            [reagent.core :as reagent]))
+
+(def tabs-data
+  [{:id :recent :label (i18n/label :t/recent) :accessibility-label :recent-tab}
+   {:id :saved :label (i18n/label :t/saved) :accessibility-label :saved-tab}
+   {:id :contacts :label (i18n/label :t/contacts) :accessibility-label :contacts-tab}
+   {:id :my-accounts :label (i18n/label :t/my-accounts) :accessibility-label :my-accounts-tab}])
+
+(defn- tab-view
+  [selected-tab]
+  (case selected-tab
+    :recent      [quo/empty-state
+                  {:title           (i18n/label :t/no-recent-transactions)
+                   :description     (i18n/label :t/make-one-it-is-easy-we-promise)
+                   :placeholder?    true
+                   :container-style style/empty-container-style}]
+    :saved       [quo/empty-state
+                  {:title           (i18n/label :t/no-saved-addresses)
+                   :description     (i18n/label :t/you-like-to-type-43-characters)
+                   :placeholder?    true
+                   :container-style style/empty-container-style}]
+    :contacts    [quo/empty-state
+                  {:title           (i18n/label :t/no-contacts)
+                   :description     (i18n/label :t/no-contacts-description)
+                   :placeholder?    true
+                   :container-style style/empty-container-style}]
+    :my-accounts [quo/empty-state
+                  {:title           (i18n/label :t/no-other-accounts)
+                   :description     (i18n/label :t/here-is-a-cat-in-a-box-instead)
+                   :placeholder?    true
+                   :container-style style/empty-container-style}]))
+
+(defn- view-internal
+  []
+  (let [margin-top   (safe-area/get-top)
+        timer        (atom nil)
+        valid-ens?   (reagent/atom false)
+        input-value  (atom "")
+        selected-tab (reagent/atom (:id (first tabs-data)))]
+    (fn [{:keys [theme]}]
+      [rn/scroll-view
+       {:content-container-style      (style/container margin-top)
+        :keyboard-should-persist-taps :never
+        :scroll-enabled               false}
+       [quo/page-nav
+        {:icon-name           :i/close
+         :on-press            #(rf/dispatch [:navigate-back])
```

Comment:
"Constant" functions can be moved outside the renderer, otherwise the entire map passed to `quo/page-nav` will be different every time `view-internal` re-renders, which will cause Reagent to lift weight to compute `page-nav`.